src/main/java/org/codebusters/audiogeek/spotifygazer/domain/util/DtoUtils.java

```diff
@@ -27,12 +31,15 @@ public static String convertToString(Object object, List<String> reservedFields)
 
     /**
      * Converts object to string
+     *
      * @param object object to convert
      * @return object data in string
      */
     public static String convertToString(Object object) {
-        var oMapper = new ObjectMapper();
-        var map = oMapper.convertValue(object, Map.class);
+        var mapper = new ObjectMapper();
```

Comment:
So the `ObjectMapper` instance is also thread safe. So to improve performance you can register only one private static instance and use it in all conversion method